round1/leaderboard.csv

```diff
@@ -1,144 +1,144 @@
-BRPHJ_NLP3,BRPHJ_NLP,automatic,yes,https://ir.nist.gov/covidSubmit/archive/round1/BRPHJ_NLP3.pdf,,0.0597,0.1067,0.0590,0.2075,0.0384,0.4742,0.0225,0.6935,0.0081,0.0045
-CincyMedIR-run2,CincyMedIR,automatic,yes,https://ir.nist.gov/covidSubmit/archive/round1/CincyMedIR-run2.pdf,,0.0590,0.1000,0.0553,0.2042,0.0392,0.4612,0.0271,0.6658,0.1543,0.0116
-CincyMedIR-run3,CincyMedIR,automatic,no,https://ir.nist.gov/covidSubmit/archive/round1/CincyMedIR-run3.pdf,,0.0590,0.1000,0.0553,0.2042,0.0392,0.4612,0.0271,0.6658,0.1543,0.0116
-yn-r1-hierarchy,NI_CCHMC,manual,yes,https://ir.nist.gov/covidSubmit/archive/round1/yn-r1-hierarchy.pdf,,0.0294,0.0667,0.0298,0.2018,0.0229,0.4603,0.0163,0.6716,0.1881,0.0146
-ERST_QUESTION,KAROTENE_SYNAPTIQ_UMBC,automatic,no,https://ir.nist.gov/covidSubmit/archive/round1/ERST_QUESTION.pdf,,0.0283,0.0467,0.0278,0.9145,0.0214,0.9303,0.0176,0.9373,0.1002,0.0059
-CincyMedIR-run1,CincyMedIR,automatic,no,https://ir.nist.gov/covidSubmit/archive/round1/CincyMedIR-run1.pdf,,0.0256,0.0467,0.0261,0.8898,0.0241,0.8685,0.0215,0.8675,0.1601,0.0106
-SFDC-23April-run2,SFDC,automatic,no,https://ir.nist.gov/covidSubmit/archive/round1/SFDC-23April-run2.pdf,,0.0218,0.0333,0.0209,0.8416,0.0174,0.8796,0.0138,0.9121,0.3007,0.0220
-yn-r1-alltext,NI_CCHMC,manual,no,https://ir.nist.gov/covidSubmit/archive/round1/yn-r1-alltext.pdf,,0.0205,0.0333,0.0213,0.5721,0.0157,0.7224,0.0125,0.8282,0.1973,0.0139
-BRPHJ_NLP2,BRPHJ_NLP,automatic,no,https://ir.nist.gov/covidSubmit/archive/round1/BRPHJ_NLP2.pdf,,0.0138,0.0267,0.0163,0.9548,0.0156,0.9609,0.0148,0.9638,0.1840,0.0090
-lda400s5000,TM_IR_HITZ,automatic,yes,https://ir.nist.gov/covidSubmit/archive/round1/lda400s5000.pdf,,0.0102,0.0133,0.0097,0.2079,0.0068,0.4712,0.0055,0.6855,0.0098,0.0006
-yn-r1-concepttext,NI_CCHMC,manual,no,https://ir.nist.gov/covidSubmit/archive/round1/yn-r1-concepttext.pdf,,0.0099,0.0067,0.0074,0.9541,0.0103,0.9502,0.0116,0.9487,0.1755,0.0101
-ERST_PROSE,KAROTENE_SYNAPTIQ_UMBC,automatic,yes,https://ir.nist.gov/covidSubmit/archive/round1/ERST_PROSE.pdf,,0.0089,0.0267,0.0102,0.2070,0.0078,0.4717,0.0059,0.6883,0.1073,0.0026
-lda400s2000,TM_IR_HITZ,automatic,no,https://ir.nist.gov/covidSubmit/archive/round1/lda400s2000.pdf,,0.0073,0.0067,0.0076,0.5734,0.0061,0.7554,0.0058,0.8613,0.0063,0.0006
-ERST_NARRATIVE,KAROTENE_SYNAPTIQ_UMBC,automatic,no,https://ir.nist.gov/covidSubmit/archive/round1/ERST_NARRATIVE.pdf,,0.0023,0.0067,0.0033,0.9780,0.0034,0.9809,0.0037,0.9814,0.0898,0.0019
-tm_lda400,TM_IR_HITZ,automatic,no,https://ir.nist.gov/covidSubmit/archive/round1/tm_lda400.pdf,,0.0000,0.0000,0.0002,0.9859,0.0013,0.9807,0.0028,0.9809,0.1090,0.0021
+run,team,report,runtype,contributed judgments,topicformat,ndcg@10,p@5,RBP(p=.5),bpref,MAP
+run1,GUIR_S2,https://ir.nist.gov/covidSubmit/archive/round1/run1.pdf,manual,yes,,0.6844,0.7933,0.7495,0.4159,0.2663
```

Comment:
Can you change the reports to relative URLs in our own repo?

Reply:
Sure. It's done